tests/drivers/spi/spi_controller_peripheral/boards/nrf54h20dk_nrf54h20_cpuapp_fast.overlay

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	spi121_default_alt: spi121_default_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 7, 2)>,
+				<NRF_PSEL(SPIM_MISO, 7, 0)>,
+				<NRF_PSEL(SPIM_MOSI, 7, 1)>;
+		};
+	};
+
```

Comment:
question: isn't there any "default" set of pins for SPI on the board (a set that is commonly used). Saying this to avoid unnecessary boilerplate in all overlays.

Reply:
the ones used in overlays are matching with what we have in CI setup, but I wouldn't say it is default setting - this is a setup that maximizes amount of peripherals that can be checked with available pin headers on nRF54H20 DK